docs/resources/api_key.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_api_key Resource - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_api_key represents a Spacelift API Key - a credential that can be used to authenticate with the Spacelift API from outside of Spacelift, typically for automation purposes.
+  WARNING
+  ** This resource manages API keys which are sensitive credentials. These keys will be saved to your state file. Ensure that you handle your state file securely. **
+---
+
+# spacelift_api_key (Resource)
+
+`spacelift_api_key` represents a Spacelift API Key - a credential that can be used to authenticate with the Spacelift API from outside of Spacelift, typically for automation purposes.
+
+### WARNING
+
+** This resource manages API keys which are sensitive credentials. These keys will be saved to your state file. Ensure that you handle your state file securely. **
```

Comment:
@b4k3r @peterdeme here is the warning we discussed internally.

Reply:
looks good, just remove the whitespaces surrounding the `**`, as it won't render properly